spec/lib/scihist_digicoll/helpers/subject_creator_adjuster_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+require "#{Rails.root}/lib/scihist_digicoll/task_helpers/subject_creator_adjuster"
+
+puts ScihistDigicoll::TaskHelpers::SubjectCreatorAdjuster
+
+describe ScihistDigicoll::TaskHelpers::SubjectCreatorAdjuster, type: :model do
+
+  let(:work_1) do 
```

Comment:
In general `work_1`, `work_2` etc are not great names in a spec. 

What makes these works different from each other  or notable or why are we even testing more than one? `work_with_x` would be a better name. 

Or do we even need more than one work to test, does one work suffice? 

Alternately, if you are using a different work in each test, it's best to instead just put each test in it's own `describe` block, that has it's own `let(:work)` scoped to that describe block. Rather than create a whole bunch of works at the top, and then only use one in each test, and then you have to look back and forth from top to the test. 

Sorry, I'm trying to pay more attention to reviewing specs as code too, after we ran into all that technical debt from the existing specs not being as careful or maintainable as they could be!

Reply:
One work is totally fine!